files/common/lib/sysctl.d/50-override.conf

```diff
@@ -93,3 +93,10 @@ fs.mount-max = 300000
 # value of 8192, resulting in upgrade failures.
 #
 fs.inotify.max_user_watches = 32768
+
+#
+# We've seen cases where an NFSv4 delegation is revoked by the server,
```

Comment:
I think, based on our tests, it was not clear if it's related to v 4.1 or just general v4 and above problem, right? If it is a client nfs version specific issue, wondering if we should just scope it 4.1 (which 6.10 uses) and leave it the default for other clients.  Not sure if we can even do that specific version based thing.
May be we should test on a platform other than Linux - Solaris x86 or HP just to make sure things are ok?

Reply:
I don't know of a way to disable NFS delegations, outside of this tunable.. and this tunable is global.. so, while I wouldn't be opposed to set this on a specific NFS version, or specific client connect, I'm not sure it's possible to do that (currently, at least).

> May be we should test on a platform other than Linux - Solaris x86 or HP just to make sure things are ok?

Sure, how do I do this?

Reply:
From blackbox selfservice, 
- pick "oracle_mt_sanity" from the drop down for test scope and in the presented OS/platform configs, pick `ORACLE_19.16.0.0.0 | Solaris 11.4 | ASM | MT` as the test config.
- in addition, probably a good idea to run Oracle RAC as well. For this, from selfservice, pick "rac_mt_sanity" for test scope and pick `Oracle 12.2.0.1.0 | CENTOS 7.4 | CLONED | RAC(DBOMSRD914EA)` as the test config.